### PR TITLE
Added navigation and modal component

### PR DESCRIPTION
**Pull request by vrinda-mahajan**
I have added the navigation and modal component to my documentation. Please review my PR.
Preview:
navigation: https://jovial-haibt-d226d3.netlify.app/docs#Navigation
modal: https://jovial-haibt-d226d3.netlify.app/docs#Modal

### CODE REVIEW

**Review by Maianki — APPROVED** [review]
Looks good to merge.

**Review by VLeads — APPROVED** [review]
Code is clean. go ahead !!

**Review by ParthParolekar — APPROVED** [review]
Looks good!

File: src/components/modal.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    z-index: 3;
+    top:0;
+    left: 0;
+    background-color: rgba(0,0,0,0.5);  
+}
+.modal-close-btn {
+    background: inherit;
+    border: none;
+    cursor: pointer;
+    font-size: 2.4rem;
```
Comment (anik31) [inline src/components/modal.css:20]: Add some space after every css selector. Everything is good. Nice work.

**Review by anik31 — APPROVED** [review]

File: src/components/modal.css (hunk 1 above)
Comment (vrinda-mahajan) [reply src/components/modal.css:20]: I will add. Thanks for reviewing.

**Pull request merged**